ext/openssl/openssl.stub.php

```diff
@@ -443,7 +443,7 @@ function openssl_x509_checkpurpose(OpenSSLCertificate|string $certificate, int $
 
 function openssl_x509_read(OpenSSLCertificate|string $certificate): OpenSSLCertificate|false {}
 
-/** @deprecated */
+#[\Deprecated]
 function openssl_x509_free(OpenSSLCertificate $certificate): void {}
```

Comment:
These OpenSSL functions are deprecated since 8.0 when the resources were converted to opaque objects and are freed automatically via the destructor.